src/workerd/server/tests/python/import_tests.bzl

```diff
@@ -31,12 +31,34 @@ const unitTests :Workerd.Config = (
 def generate_wd_test_file(requirement):
     return WD_FILE_TEMPLATE.format(requirement, requirement)
 
+SKIP_ON_0_27_1 = [
+    "PyJWT",
+    "beautifulsoup4",
+    "cffi",
+    "cryptography",
+    "pycparser",
+    "python-multipart",
+    "soupsieve",
+    "sqlalchemy",
+    "starlette",
+    "tblib",
+]
```

Comment:
Can we move this into BUILD.bazel and not duplicate the mechanism for skipping? I think it makes sense to organize them like this rather than 
```
            "micropip": [
                "0.27.1",
            ],
```
which is more verbose.

Reply:
Ahh sorry, forgot about this mechanism. Done!